config.example.toml

```diff
@@ -3,4 +3,5 @@ endpoint = "http://0.0.0.0:8899"
 
 [grpc]
 endpoint = "http://0.0.0.0:10000"
+x-token = "<X-TOKEN>"
```

Comment:
```suggestion
x_token = "<X-TOKEN>"
```

Reply:
This should be a dash as it is in the PR. it gets converted to underscore by serde.